srcpkgs/ReDoomEd.app/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'ReDoomEd.app'
+pkgname=ReDoomEd.app
+version=0.92.1.b1
+revision=1
+wrksrc="ReDoomEd.Sources.${version%.*}-${version##*.}"
+build_wrksrc=ReDoomEd
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc tar which"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
```

Comment:
These aren't needed, it detects the dependencies.

Reply:
wait, how does it do that?

Reply:
It scans the files for libraries it depends on, looks up the package in common/shlibs and then adds that automatically to the dependency list.

Reply:
I tried this and this is what broke the program building in reality, not the if statement

Reply:
I found out I can omit dependencies\, but not build dependencies except for now-dumb ones like tar and which

Reply:
I also found out make_use_env=yes was needed